samples/README.md

```diff
@@ -13,7 +14,81 @@ curl -O -fsSL https://raw.github.com/ppc64le-cloud/pvsadm/master/samples/image-c
 curl -O -fsSL https://raw.github.com/ppc64le-cloud/pvsadm/master/samples/image-create-and-upload-sample/config.sh
 ```
 2. Change values in the config.sh file as per description given in file
-3. Run the script 
+3. Run the script
 ```
 ./image_create_and_upload_script.sh
-```
\ No newline at end of file
+```
+
+### Script 2: Sample script for RHEL/RHCOS/CENTOS images in PowerVS service from a Centos VM.
+The convert-upload-images-powervs script helps to create RHEL/RHCOS/CENTOS images in PowerVS services. This script creates its own CentOS vm to do all the conversion tasks.
+
+#### What this script does - High Level Steps
+1. Gets CentOS OVA image from IBM FTP server or Unicamp
```

Comment:
Do we have the URL handy or specified somewhere else?

Reply:
as of now, there isn't a URL pointing to 250G image